### PR TITLE
Remove execute() function

### PR DESCRIPTION
**Pull request by 1ucian0**
### Summary

Removal of `execute` which was deprecated in 0.46 https://github.com/Qiskit/qiskit/pull/11044

### CODE REVIEW

File: examples/python/using_qiskit_terra_level_0.py
Comment (ElePT) [inline examples/python/using_qiskit_terra_level_0.py]: I don't think that "level 0" referred to the optimization level, I think that this was intended as a beginner-level example (maybe remove the mention to level 0 altogether?)

File: examples/python/using_qiskit_terra_level_0.py
Comment (ElePT) [inline examples/python/using_qiskit_terra_level_0.py]:
```suggestion
This example shows the most basic way to use Qiskit. It builds some circuits
```

**Review by ElePT — COMMENTED** [review]
LGTM (once the lint/docs complaints are fixed). I anticipate some conflicts with https://github.com/Qiskit/qiskit/pull/11422 just because we modify the same files, but they should be relatively easy to handle.

File: examples/python/using_qiskit_terra_level_0.py
Comment (1ucian0) [reply examples/python/using_qiskit_terra_level_0.py]: oooohh... that totally makes sense. I changed for `how to use Qiskit at introduction level.` in 2821c487daf2ad868a892a7ed5222b45216b1bf2

**Review by ElePT — DISMISSED** [review]
Happy to re-approve once the merge conflict is fixed.

**Review by ElePT — DISMISSED** [review]

**Comment by ElePT** [comment]
@1ucian0 checking the docs build logs, I've realized that the API docs entry for execute needs to be removed too, that is `docs/apidocs/execute.rst`+ the execute entry in `docs/apidocs/index.rst`. I think that the entry is ignored if the class is not found and the docs still build without errors, and that's why it has slipped under the radar.

**Comment by 1ucian0** [comment]
good catch! fixing it in 7c5fc5e243b616c43c077510ad8cc3200e799648

**Review by ElePT — APPROVED** [review]
I looks like the docs build has finally succeeded. Let's hope that there are no further surprises.

**Pull request merged**
